lib/Sema/DerivedConformanceCodable.cpp

```diff
@@ -357,8 +356,8 @@ static EnumDecl *synthesizeCodingKeysEnum(DerivedConformance &derived) {
   // Each of these vars needs a case in the enum. For each var decl, if the type
   // conforms to {En,De}codable, add it to the enum.
   bool allConform = true;
-  for (auto *varDecl : target->getStoredProperties(/*skipInaccessible=*/true)) {
-    if (varDecl->getAttrs().hasAttribute<LazyAttr>())
+  for (auto *varDecl : target->getStoredProperties()) {
+    if (!varDecl->isUserAccessible())
```

Comment:
Yeah, it might be nice to define it directly at some point (right now I think the only decls with this bit set are backing storage for lazy and wrappers).

isInvalid() is another silly one too that we should replace with something derived from the other properties of the decl.